features/restaurant_edit_details.feature

```diff
@@ -0,0 +1,21 @@
+Feature: As a system user
+  In order to keep information about my restaurant up to date
+  I would like to be able to edit my restaurant's details
+
+  Background:
+    Given I am logged in as a user of the system
+    And the following restaurant exists
+    | id | name    | address     | zip_code |
+    | 1  | Mikey's | Main Street | 12345    |
```

Comment:
Also since you don't seem to use `address` && `zip_code` in your tests you can safely remove them from the table. You should only keep the info that is unique for the instance and let FactoryGirl do the rest of the job.